genienlp/tasks/generic_task.py

```diff
@@ -253,8 +253,65 @@ def get_splits(self, root, **kwargs):
         return OODDataset.splits(root=root, **kwargs)
 
 
+class E2EDialogueTask(BaseTask):
```

Comment:
I think this is time to move dialogue tasks and datasets to separate files named `dialogue_task.py` and `dialogue_dataset.py`

Reply:
Sure